breathecode/feedback/models.py

```diff
@@ -99,6 +99,7 @@ def __init__(self, *args, **kwargs):
     highest = models.CharField(max_length=50, default="very likely")
     lang = models.CharField(max_length=3, blank=True, default="en")
 
+    platform_and_content = models.BooleanField(default=False)
```

Comment:
This solution is a bit inaccurate https://github.com/breatheco-de/breatheco-de/issues/7574#issuecomment-2249049862